Source/Core/Core/AchievementManager.cpp

```diff
@@ -38,6 +38,9 @@ void AchievementManager::Init()
 {
   if (!m_is_runtime_initialized && Config::Get(Config::RA_ENABLED))
   {
+    std::string host_url = Config::Get(Config::RA_HOST_URL);
+    if (!host_url.empty())
+      rc_api_set_host(host_url.c_str());
```

Comment:
Just wondering, does this properly handle cases where the entered URL is not a valid achievement endpoint (or otherwise unavailable/unaccessible)?

Reply:
You'll get an error in the logger and otherwise it will function basically identically to just not having a connection at all.